docs/content/doc/installation/from-binary.en-us.md

```diff
@@ -24,13 +24,38 @@ embedded assets. This can be different for older releases.
 
 ## Download
 
-Choose the file matching your platform from the [downloads page](https://dl.gitea.io/gitea/), copy the URL and replace the URL within the commands below:
+You can find the file matching your platform from the [downloads page](https://dl.gitea.io/gitea/) after navigating to the version you want to download. 
+
+### Choosing the right file
+For Linux, you should choose: 
+ * `linux-386` for 32-bit Intel/AMD processors
```

Comment:
This isn't quite right. It's for 32bit operating systems. If you have a 64bit CPU but are still running a 32bit OS you will still need this linux-386 build.